src/index.html

```diff
@@ -1,16 +1,56 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
+      method="get"
+      class="search-form"
+      data-qa="big"
+    >
+      <label>
+        <input
+          class="search-input search-input--big"
```

Comment:
```suggestion
          class="search-input"
```

Reply:
redundant. make styles for search-input and just change necessary property for search-input-small

Reply:
![Screenshot from 2023-04-04 15-20-12](https://user-images.githubusercontent.com/33029864/229798714-1c9397f7-3cc5-4f9c-8731-f9b8bf536e74.png)